book/developing/overview.md

```
@@ -1,5 +1,59 @@
+## Getting Started
+
+### Clone the repo
+```
+git clone https://github.com/andrewdavidmackenzie/flow.git
+```
+
+### Install build tools
+You need `make` and a Rust toolchain (I suggest [rustup](https://rustup.rs/)).
+Once you have those, install the remaining dependencies with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)